packages/frontend/src/components/Map/types.ts

```diff
@@ -0,0 +1,9 @@
+import { Feature } from 'geojson';
+
+export interface FinlandMapFeature extends Feature {
+  properties: {
+    id: number;
+    kpalue: number;
+    tilirataos: number;
```

Comment:
As previously, these should be translated. Though I don't quite know what the translations would be. Any idea @saneDG? 🤔

Reply:
These are tricky.. 
kpalue -> maintenanceArea
tilirataos(tilirataosa??) -> ???

Reply:
This is a bit tricky, Finnish keys represent original Json data

Reply:
Right. In that case I'd say this is fine then.